pkg/distro/rhel9/images.go

```diff
@@ -299,6 +299,18 @@ func edgeCommitImage(workload workload.Workload,
 	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || t.arch.distro.osVersion == "9-stream" {
 		img.OSCustomizations.EnabledServices = append(img.OSCustomizations.EnabledServices, "ignition-firstboot-complete.service", "coreos-ignition-write-issues.service")
 	}
+	img.OSCustomizations.SystemdUnit = []*osbuild.SystemdUnitStageOptions{
+		{
+			Unit:     "grub-boot-success.timer",
+			Dropin:   "10-disable-if-greenboot.conf",
+			UnitType: osbuild.Global,
+			Config: osbuild.SystemdServiceUnitDropin{
+				Unit: &osbuild.SystemdUnitSection{
+					FileExists: "!/usr/libexec/greenboot/greenboot",
+				},
+			},
+		},
+	}
```

Comment:
@thozza I get the ImageConfig, `where to the define the above dropin` which I think you referred as

> be set in the image definition itself

 so that I can refer that in the commitImage and  edgeContainerImage